spiceaidocs/content/en/reference/Postgres/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Postgres"
+linkTitle: "Postgres"
+description: 'PostgreSQL reference'
+weight: 80
+---
+
+PostgreSQL can be connected to Spice AI as either a dataset source, a data store, or a data mesh.
+
+## Dataset Source
+
+To use PostgreSQL as a dataset source, specify `postgres` as the `source` portion of `from` value for your dataset.
+
+```yaml
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+        enabled: true
+```
+
+## Data Mesh
+
+To use PostgreSQL as a data mesh, specify `postgres` as the `source` portion of `from` value for your dataset, without acceleration.
+
+```yaml
+datasets:
+  - from: postgres:path.to.my_dataset
+    name: my_dataset
+```
+
+## Data Store
+
+To use PostgreSQL as a data store, specify `postgres` as the `engine` for your dataset.
+
+```yaml
+datasets:
+  - from: spiceai:path.to.my_dataset
+    name: my_dataset
+    acceleration:
+        engine: postgres
+```
+
+## Configuration
+
+You can configure the connection to your PostgreSQL by providing the following `params`:
```

Comment:
Remove "you" and "your". E.g. The PostgreSQL connection is configured with the following parameters.